src/devtools/client/webconsole/components/Output/ConsoleOutput.tsx

```diff
@@ -232,20 +231,18 @@ class ConsoleOutput extends React.Component<PropsFromRedux, State> {
         </div>
         {contextMenu !== null && (
           <ContextMenu x={contextMenu.pageX} y={contextMenu.pageY} close={this.closeContextMenu}>
-            <Dropdown>
-              <DropdownItem onClick={this.setFocusStart}>
-                <>
-                  <Icon filename="set-focus-start" className="mr-4 bg-iconColor" size="large" />
-                  Set focus start
-                </>
-              </DropdownItem>
-              <DropdownItem onClick={this.setFocusEnd}>
-                <>
-                  <Icon filename="set-focus-end" className="mr-4 bg-iconColor" size="large" />
-                  Set focus end
-                </>
-              </DropdownItem>
-            </Dropdown>
+            <ContextMenuItem onClick={this.setFocusStart}>
+              <>
+                <Icon filename="set-focus-start" className="mr-4 bg-iconColor" size="large" />
+                Set focus start
+              </>
+            </ContextMenuItem>
+            <ContextMenuItem onClick={this.setFocusEnd}>
+              <>
+                <Icon filename="set-focus-end" className="mr-4 bg-iconColor" size="large" />
+                Set focus end
+              </>
+            </ContextMenuItem>
```

Comment:
Keyboard navigation doesn't work in this case, (or the one in Network monitor). Do you know why?